packages/DApp/src/components/card/CardVote/CardVote.tsx

```diff
@@ -50,11 +55,28 @@ export const CardVote = ({ room, hideModalFunction }: CardVoteProps) => {
     setShowConfirmModal(val)
   }
 
-  const verificationPeriod =
-    room.verificationStartAt.toNumber() * 1000 - Date.now() < 0 && room.endAt.toNumber() * 1000 - Date.now() > 0
+  const now = Date.now()
+  const verificationStarted = room.verificationStartAt.toNumber() * 1000 - now < 0
+  const verificationEnded = room.endAt.toNumber() * 1000 - now < 0
+  const verificationPeriod = verificationStarted && !verificationEnded
 
   const winner = verificationPeriod ? 0 : getVotingWinner(vote)
 
+  const { votesFor: votesForUnverified, votesAgainst: votesAgainstUnverified } = useUnverifiedVotes(
+    vote.ID,
+    room.verificationStartAt,
+    room.startAt
+  )
+
+  const includeUnverifiedVotes = !winner || verificationPeriod
+
+  const votesFor = !includeUnverifiedVotes
+    ? vote.voteFor.toNumber()
+    : vote.voteFor.toNumber() + votesForUnverified + sentVotesFor
+  const votesAgainst = !includeUnverifiedVotes
+    ? vote.voteAgainst.toNumber()
+    : vote.voteAgainst.toNumber() + votesAgainstUnverified + sentVotesAgainst
```

Comment:
During verification period, and whenever votes are verified thus aded to `vote.voteFor` or `vote.voteAgaints`, I believe , the total amount could be inflated and include the same vote twice (i.e. verified and unverified).

Reply:
@jkbktl do you think we should reduce the votes in this case too?

Reply:
I think if I am not mistaken that's the reason why there was condition "already voted", because in initial phase it adds addresses which already voted and are verified, therefore they should not be counted as unverified.

Reply:
Reverted at https://github.com/status-im/community-dapp/pull/31/commits/967d4f04e124c7f7057d565bed7b96f16ee6730b.